docs/debugging.md

```diff
@@ -14,97 +14,46 @@ React Native provides an in-app developer menu which offers several debugging op
 
 Alternatively for Android devices and emulators, you can run `adb shell input keyevent 82` in your terminal.
 
-![The React Native Dev Menu](/docs/assets/debugging-dev-menu.jpg)
+![The React Native Dev Menu](/docs/assets/debugging-dev-menu-076.jpg)
 
 :::note
 The Dev Menu is disabled in release (production) builds.
 :::
 
-## Opening the Debugger
+## Opening DevTools
 
-The debugger allows you to understand and debug how your JavaScript code is running, similar to a web browser.
+[React Native DevTools](./react-native-devtools) is our included debugger for React Native. It allows you to inspect and understand how your JavaScript code is running, similar to a web browser.
```

Comment:
```suggestion
[React Native DevTools](./react-native-devtools) is our debugger for React Native. It allows you to inspect and understand how your JavaScript code is running, similar to a web browser.
```

Reply:
Can we keep this? In one adjective, we've reinforced that this comes out-of-the box, with no separate "how do I install this?" step.

Reply:
I'd be more precise then... "is our debugger for React Native, shipped with React Native" or something like that. "Included" put there felt a bit confusing (included where?).

wdyt?

Reply:
I think this one reads clear enough, hoping to keep this first sentence pair as brief/flowy as possible. "built-in" is a possible substitute.

Reply:
"built-in" is better than "shipped"! 😄